cli/go.mod

```diff
@@ -5,7 +5,7 @@ go 1.18
 require (
 	github.com/AlecAivazis/survey/v2 v2.3.5
 	github.com/DataDog/zstd v1.5.2
-	github.com/Masterminds/semver v1.5.0
```

Comment:
Super lazy of me not to google this, but what's the mechanism to update the installed package when you want to do this? Is it just `go get github.com/Masterminds/semver`?

Reply:
Go gets funky with major version bumps so you need to do `go get github.com/Masterminds/semver/v3` and it's viewed as a new package. If it was a non-major version change you can just do `go get github.com/Masterminds/semver@1.6.0`